### PR TITLE
Update bottom camera topic

### PR DESCRIPTION
**Pull request by bitoffdev**

### CODE REVIEW

**Review by aftersomemath — APPROVED** [review]

File: src/iarc7_vision/roomba_blob_vision_node.py [hunk 1]
```diff
@@ -221,6 +222,13 @@ class CameraProcessor(ImageRoombaFinder):
     def __init__(self, debugging_on):
         rospy.init_node(NODE_NAME, anonymous=True)
 
+        # Verify that the correct version of OpenCV is in use
+        if cv2.__version__[:3] != "2.4":
+            rospy.logerr("You are using the incorrect version of OpenCV. You " +
+                         "should be using version 2.4, but you are using " +
```
Comment (aftersomemath) [inline src/iarc7_vision/roomba_blob_vision_node.py]: We need to be using 2.4.13 specifically

File: src/iarc7_vision/roomba_blob_vision_node.py [hunk 2]
```diff
@@ -344,7 +352,13 @@ def __init__(self, file_path):
         """
         :param file_path: path to video file
         """
-        super(VideoProcessor, self).__init__()
+        # Verify that the correct version of OpenCV is in use
+        if cv2.__version__[:3] != "2.4":
+            print("You are using the incorrect version of OpenCV. You " +
+                         "should be using version 2.4, but you are using " +
```
Comment (aftersomemath) [inline src/iarc7_vision/roomba_blob_vision_node.py]: We don't need to do the check twice.

Comment (bitoffdev) [reply src/iarc7_vision/roomba_blob_vision_node.py]: They are in different classes. The CameraProcessor class relies on ROS, so it logs the error to the rospy output. The VideoProcessor class does not depend on ROS, so it logs the error to standard output.

Comment (aftersomemath) [reply src/iarc7_vision/roomba_blob_vision_node.py]: Why don't you do it under line 372 and just print to standard output?

**Review by aftersomemath — APPROVED** [review]

**Pull request merged**